OpenEXR/IlmImf/ImfPizCompressor.cpp

```diff
@@ -183,8 +183,9 @@ PizCompressor::PizCompressor
     _channels (hdr.channels()),
     _channelData (0)
 {
-    size_t tmpBufferSize =
-                uiMult (maxScanLineSize, numScanLines) / 2;
+    // TODO: Remove this when we can change the ABI
+    (void) _maxScanLineSize;
```

Comment:
Why the (void) _maxScanLineSize; ?

Reply:
Because I could "fake" the member being used without further warnings (the (void) cast bit is how that is done), but if I just removed the member, I would change the vtable layout, which I didn't want to do, I thought we were trying to avoid any of that...